_code-samples/account-configurator/account-configurator.js

```diff
@@ -0,0 +1,478 @@
+  document.getElementById("allowTrustLineClawback").checked = parsedResults.result.account_flags.allowTrustLineClawback
+  document.getElementById("depositAuth").checked = parsedResults.result.account_flags.depositAuth
+  document.getElementById("disableMasterKey").checked = parsedResults.result.account_flags.disableMasterKey
+  document.getElementById("disallowIncomingCheck").checked = parsedResults.result.account_flags.disallowIncomingCheck
+  document.getElementById("disallowIncomingNFTokenOffer").checked = parsedResults.result.account_flags.disallowIncomingNFTokenOffer
+  document.getElementById("disallowIncomingPayChan").checked = parsedResults.result.account_flags.disallowIncomingPayChan
+  document.getElementById("disallowIncomingTrustline").checked = parsedResults.result.account_flags.disallowIncomingTrustline
+  document.getElementById("disallowIncomingXRP").checked = parsedResults.result.account_flags.disallowIncomingXRP
+  document.getElementById("globalFreeze").checked = parsedResults.result.account_flags.globalFreeze
+  document.getElementById("noFreeze").checked = parsedResults.result.account_flags.noFreeze
+  document.getElementById("requireAuthorization").checked = parsedResults.result.account_flags.requireAuthorization
+  document.getElementById("requireDestinationTag").checked = parsedResults.result.account_flags.requireDestinationTag
+  try {
+    let signerEntries_variable = parsedResults.result.account_data.signer_lists[0].SignerEntries
+      document.getElementById("signer1AccountField").value = signerEntries_variable[0]["SignerEntry"]["Account"]
+      document.getElementById("signer1WeightField").value = signerEntries_variable[0]["SignerEntry"]["SignerWeight"]
+      document.getElementById("signer2AccountField").value = signerEntries_variable[1]["SignerEntry"]["Account"]
+      document.getElementById("signer2WeightField").value = signerEntries_variable[1]["SignerEntry"]["SignerWeight"]
+      document.getElementById("signer3AccountField").value = signerEntries_variable[2]["SignerEntry"]["Account"]
+      document.getElementById("signer3WeightField").value = signerEntries_variable[2]["SignerEntry"]["SignerWeight"]
+      document.getElementById("signerQuorumField").value = parsedResults.result.account_data.signer_lists[0].SignerQuorum
+  } catch (error) {
+  }
+  client.disconnect()
+} // End of getAccountInfo()
+
+// *******************************************************
+// **************** Configure Account ********************
+// *******************************************************
+
+async function configureAccount() {
+  let net = getNet()
+  const client = new xrpl.Client(net)
+  resultField.value = `Connecting to ${net}.`
+  await client.connect()
+  my_wallet = xrpl.Wallet.fromSeed(seedField.value)
+  const my_acct_info = await client.request({
+    command: "account_info",
+    account: accountField.value,
+    ledger_index: "validated",
+  })
+  // client.disconnect()
```

Comment:
How come this is commented out?

Reply:
Because it was unnecessary, silly. Why wasn't it removed? Er...uh....